### ISSUE TITLE
Trouble downloading finished Prosit Data

### ISSUE THREAD

**Issue by jacoblippincott** [kusterlab/prosit#28]
What happened: I started a Prosit search by uploading a CSV file. The file processed to completion and I was able to download the finished zip file. However, after downloading 1GB of 1.4GB, the download said it was finished. I tried to unzip the file but I got an error saying the data was corrupted. 

What should have happened: The whole zip file should have downloaded.

Steps to reproduce: Here is the link to the data: https://www.proteomicsdb.org/prosit/task/7B5DCF81B8D4F1945C9FE82B5473177F. Try downloading this data. The 1st gig will download and then it will stop. I have tried to process this dataset two times now and am in the process of running it again. I can provide the CSV file if needed.

**Comment by tkschmidt** [comment]
Hey,
thank you for using our tool and sry for the inconvenience. We had a major change in our backend architecture and I set wrong maximal download sizes. It should be fixed now as I was able to download your job.
Best,
Tobias

**Comment by jacoblippincott** [comment]
Hello Tobias,



Thank you for fixing this so quickly. I tired the download myself and it
worked just fine. We here at Proteome Software are big fans of Prosit.



Jacob



*From:* Tobias <notifications@github.com>
*Sent:* Wednesday, November 27, 2019 6:30 AM
*To:* kusterlab/prosit <prosit@noreply.github.com>
*Cc:* jacoblippincott <jacob@proteomesoftware.com>; Author <
author@noreply.github.com>
*Subject:* Re: [kusterlab/prosit] Trouble downloading finished Prosit Data
(#28)



Hey,
thank you for using our tool and sry for the inconvenience. We had a major
change in our backend architecture and I set wrong maximal download sizes.
It should be fixed now as I was able to download your job.
Best,
Tobias

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/kusterlab/prosit/issues/28?email_source=notifications&email_token=AN4IXUKRRAESAZV6Q52NEWDQVZ75ZA5CNFSM4JR4S7V2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEFJVTDA#issuecomment-559110540>,
or unsubscribe
